examples/github/build-and-deploy-mender-artifact.yml

```diff
@@ -74,23 +74,23 @@ jobs:
           MENDER_DEPLOYMENT_NAME=${MENDER_RELEASE_NAME}_${COMMIT_SHORT_SHA}
           echo "mender_deployment_name=${MENDER_DEPLOYMENT_NAME}" >> ${GITHUB_OUTPUT}
       # Creates Mender Artifact, using randomly generated dummy name
+      # In this example, hello-mender.py represents a single file Python application that we have
+      # built and test in our CI and now we want to deploy into a device running Mender. We will
+      # package it as a Single File Application Update. For more information visit Mender docs:
+      # https://docs.mender.io/artifact-creation/create-an-artifact
       - name: Create Mender Artifact
         id: create_artifact
         run: |
+          curl -O https://raw.githubusercontent.com/mendersoftware/mender-ci-workflows/master/examples/hello-mender.py
           MENDER_ARTIFACT_NAME=artifact_${MENDER_RELEASE_NAME}.mender
-          echo /home/root/.ssh > dest_dir
-          echo authorized_keys > filename
-          echo key > authorized_keys
-          mender-artifact write module-image \
-            -T single-file \
+          single-file-artifact-gen \
             --device-type raspberrypi4 \
             -o ${MENDER_ARTIFACTS_CI_ARTIFACT_PATH}/${MENDER_ARTIFACT_NAME} \
             -n ${MENDER_RELEASE_NAME} \
-            --software-name authorized_keys \
-            --software-version 1.0, \
-            -f dest_dir \
-            -f filename \
-            -f authorized_keys
+            --software-version 1.0 \
+            --software-filesystem rootfs \
```

Comment:
```suggestion
            --software-version 1.0 \
            --software-name hello-mender \
            --software-filesystem rootfs \
```

Reply:
My bad, I thought that `--software-name` was being set by the generator(s). Thanks, I'll fix everywhere :+1:

Reply:
I am removing also `--software-filesystem` and let `mender-artifact` use the default.